pkg/controller/datadogagent/clusteragent.go

```diff
@@ -1070,25 +1075,25 @@ func buildClusterAgentClusterRole(dda *datadoghqv1alpha1.DatadogAgent, name, age
 				datadoghqv1alpha1.UpdateVerb},
 		})
 
-		// Replicasets
+		// ExtendedDaemonsetReplicaSets
```

Comment:
do we need this?

Reply:
Yes for pod, managed by an extendeddaemonset

Reply:
it is to be futur proof :)